#Parameter server configuration for bta_ros.
#TODO change to multicast before publishing!!!
udpDataIpAddr: {
n1: 192,
n2: 168,
n3: 0,
n4: 2,
}

udpDataIpAddrLen: 4
udpDataPort: 10002
tcpDeviceIpAddr: {
n1: 192,
n2: 168,
n3: 0,
n4: 10,
}

tcpDeviceIpAddrLen: 4
tcpControlPort: 10001

#udpControlOutIpAddr: 
#udpControlOutIpAddrLen:
#udpControlOutPort:
#udpControlInIpAddr:
#udpControlInIpAddrLen:
#udpControlInPort: 
#tcpDataPort:
#uartPortName:
#uartBaudRate:
#uartDataBits:
#uartStopBits:
#uartParity:
#uartTransmitterAddress:
#uartReceiverAddress
#serialNumber:
#calibFileName:
#frameMode:
#infoEvent:
#frameArrived:
#frameQueueMode:
verbosity: 5
frameQueueLength: 1

# Optional parameter.
#frameRate: 15
#integrationTime: 1500
